wolfhsm/wh_comm.h

```diff
@@ -34,25 +34,37 @@
  * enumerated either within wolfhsm/wh_error.h.  Unless otherwise noted, all
  * functions are non-blocking and each may update the context state or perform
  * other bookkeeping actions as necessary.
+ *
+ * Configuration values:
+ *  WOLFHSM_CFG_COMM_DATA_LEN - Maximum length of data payload
+ *      Default: 1280 bytes
+ *
  */
 
 #ifndef WOLFHSM_WH_COMM_H_
 #define WOLFHSM_WH_COMM_H_
 
+#ifdef WOLFSSL_USER_SETTINGS
+#include "user_settings.h"
+#endif
+
 #include <stdint.h>  /* For sized ints */
 
 /** Packet content types */
 
 /* Request/response packets are composed of a single fixed-length header
- * (whHeader) followed immediately by variable-length data between 0 and
- * DATA_LEN bytes.
+ * (whCommHeader) followed immediately by variable-length data between 0 and
+ * WOLFHSM_CFG_COMM_DATA_LEN bytes.
  */
-enum WH_COMM_ENUM {
-    WH_COMM_HEADER_LEN = 8,    /* whCommHeader */
-    WH_COMM_DATA_LEN = 1280,
-    WH_COMM_MTU = (WH_COMM_HEADER_LEN + WH_COMM_DATA_LEN),
-    WH_COMM_MTU_U64_COUNT = (WH_COMM_MTU + 7) / 8,  /* internal U64 buffer */
-};
+#ifndef WOLFHSM_CFG_COMM_DATA_LEN
+#define WOLFHSM_CFG_COMM_DATA_LEN 1280
+#endif
+
+/* III Previous name.  Update */
+#define WH_COMM_DATA_LEN WOLFHSM_CFG_COMM_DATA_LEN
```

Comment:
lingering TODO?

Reply:
oops!  Missed it.  I'll go back and fix it in the tests and examples too.